src/systemctl/systemctl.c

```diff
@@ -1658,6 +1658,9 @@ static int list_unit_files(int argc, char *argv[], void *userdata) {
                 for (unit = units; unit < units + c; unit++)
                         free(unit->path);
 
+        if (c == 0)
+            return 1;
```

Comment:
8 space indent please

Reply:
Sorry, this is embarrassing. Pushed a change which makes this line use proper indentation.